h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -722,4 +722,17 @@ private static synchronized boolean hasGPU_impl(int gpu_id) {
     warn("_max_runtime_secs", "Disabling maximum allowed runtime for cross-validation main model.");
   }
 
+  @Override
+  public Model.Parameters.CategoricalEncodingScheme[] supportedCategoricalEncodingSchemes() {
+    return new Model.Parameters.CategoricalEncodingScheme[] {
```

Comment:
Fresh copy is returned each time, which is nice if we want to stick with arrays. Great.